platforms/CommunityToolkit.Labs.Wasm/CommunityToolkit.Labs.Wasm.csproj

```diff
@@ -34,4 +34,8 @@
     <EmbeddedResource Include="WasmCSS\Fonts.css" />
     <EmbeddedResource Include="WasmScripts\AppManifest.js" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Markdig" Version="0.30.4" />
```

Comment:
Oh, I have to add the Markdig reference to all WASM sample heads... 🤔

I need to do this in `Labs.Heads.Wasm.props` instead, oops. That's what I get for just adding the package via VS... 🙈